Source/JavaScriptCore/wasm/WasmCallee.cpp

```diff
@@ -234,6 +235,33 @@ const WasmInstruction* LLIntCallee::outOfLineJumpTarget(const WasmInstruction* p
     return m_instructions->at(offset + target).ptr();
 }
 
+void OptimizingJITCallee::addCodeOrigin(unsigned firstInlineCSI, unsigned lastInlineCSI, const Wasm::ModuleInformation& info, uint32_t functionIndex)
+{
+    // The inline frame list is stored in postorder. For example:
+    // A { B() C() D { E() } F() } -> B C E D F A
+#if ASSERT_ENABLED
+    ASSERT(firstInlineCSI <= lastInlineCSI);
+    for (unsigned i = 0; i + 1 < codeOrigins.size(); ++i)
+        ASSERT(codeOrigins[i].lastInlineCSI <= codeOrigins[i + 1].lastInlineCSI);
+    for (unsigned i = 0; i < codeOrigins.size(); ++i)
+        ASSERT(codeOrigins[i].lastInlineCSI <= lastInlineCSI);
+#endif
+    codeOrigins.append({ firstInlineCSI, lastInlineCSI, IndexOrName { functionIndex, info.nameSection->get(functionIndex) } });
```

Comment:
Let's avoid adding `IndexOrName` since it is too costly in terms of size etc. `functionIndex` is enough.